src/core/Decorations.ts

```diff
@@ -150,24 +147,24 @@ export default class Decorations {
         decoration: vscode.window.createTextEditorDecorationType({
           rangeBehavior: vscode.DecorationRangeBehavior.ClosedClosed,
           light: {
-            after: {
+            before: {
               contentIconPath: this.constructColoredSvgDataUri(
                 svg,
                 colorSetting.light
               ),
             },
           },
           dark: {
-            after: {
+            before: {
               contentIconPath: this.constructColoredSvgDataUri(
                 svg,
                 colorSetting.dark
               ),
             },
           },
-          after: {
-            margin: `-${svgHeightPx}px 0 0 -${spanWidthPx}px`,
-            width: `${spanWidthPx}px`,
+          before: {
+            margin: `-${svgHeightPx}px -${svgWidthPx}px 0 0`,
+            width: `${svgWidthPx}px`,
```

Comment:
No need to play games with span width anymore because we now align to the left side of the char